includes/multi-currency/MultiCurrency.php

```diff
@@ -351,7 +351,7 @@ public function enqueue_admin_scripts() {
 		$this->register_admin_scripts();
 
 		wp_enqueue_script( 'WCPAY_MULTI_CURRENCY_SETTINGS' );
-		wp_enqueue_style( 'WCPAY_MULTI_CURRENCY_SETTINGS' );
+		WC_Payments_Utils::register_and_enqueue_style( 'WCPAY_MULTI_CURRENCY_SETTINGS', '', [], WC_VERSION, 'all', true );
```

Comment:
We are registering the same style twice, with different params here. We could probably do something similar to [the WC `enqueue_style` method](https://github.com/woocommerce/woocommerce/blob/fd9e92e7bd4ff598a3fd8a8857ab4fd6cb2da397/plugins/woocommerce/includes/class-wc-frontend-scripts.php#L176) where it is only registered if the required params are provided, otherwise just enqueued.

Reply:
Thank you @ismaeldcom for the review! I agree and I updated the implementation.

The only thing I did differently from the linked WC method is explicitly checking for `''` inside the if: `if ( '' !== $path )` before registering the style.